MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -1087,12 +1071,26 @@ private void dispatch(BandwidthMetricData data) {
     }
   }
 
+  protected void detectStreamType(Format format) {
+    // This is reliable way to detect stream type, but it get called a bit too late,
```

Comment:
Similar to the comment earlier, is this needed?

Reply:
It would save me a lot of time when I revisit this subject.